ru_RU/webapp/src/main/webapp/templates/freemarker/termsOfUse_ru_RU.ftl

```diff
@@ -0,0 +1,14 @@
+<#-- $This file is distributed under the terms of the license in /doc/license.txt$ -->
+
+<section id="terms" role="region">
+    <h2>Правила пользования</h2>
+    
+    <h3>Ограничения ответственности</h3>
+    <p>Этот сайт ${termsOfUse.siteName} содержит материалы&mdash; текстовую информацию, ссылки на публикации, ссылки и изображения&mdash; предоставленные ${termsOfUse.siteHost} и различными третьими сторонами, как частными лицами, так и организациями, коммерческими и иными. В той мере, в какой это возможно с точки зрения авторского права, защищённая авторским правом информация, представленная на сайте VIVO и доступная в виде данных Resource Description Framework (RDF) от VIVO по адресу ${termsOfUse.siteHost}, предназначена для общественного использования и свободно распространяется на условиях лицензии <a href="http://creativecommons.org/licenses/by/3.0/" target="_blank" title="creative commons">Creative Commons CC-BY 3.0</a>, которая позволяет копировать, распространять, демонстрировать и делать производные этой информации при условии, что вы укажете ${termsOfUse.siteHost}. Любая информация, не защищенная авторским правом, доступна вам по <a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="cco waiver">CC0 waiver</a>. Однако исходные документы, изображения или веб-страницы, прикрепленные к VIVO или связанные с VIVO, могут содержать информацию, защищенную авторским правом, и должны использоваться или распространяться только на условиях, прилагаемых к каждому источнику, или в соответствии с принципами добросовестного использования.</p>
```

Comment:
Tag p is not highlighted well. I am wondering whether there is some problem in encoding or in "\&mdash;"

Reply:
Looks like some github limitation. Tags looks good to me.